mainnet/lighthouse/lighthouse.toml

```diff
@@ -6,15 +6,15 @@ subscribe-all-subnets = true
 listen-address = "0.0.0.0"
```

Comment:
How many `inbound` peers were present in test scenario?

https://github.com/sigp/lighthouse/blob/e9a7316f1de113d118f1aa16856ae97e181ceb82/beacon_node/src/cli.rs#L193 -> IDK if `0.0.0.0` is not an issue here